cmd/reconciler-manager/main.go

```diff
@@ -92,10 +93,10 @@ func main() {
 		setupLog.Error(err, "failed to build dynamic client")
 		os.Exit(1)
 	}
-	watchFleetMembership := fleetMembershipCRDExists(dynamicClient, mgr.GetRESTMapper())
+	watchFleetMembership := fleetMembershipCRDExists(dynamicClient, mgr.GetRESTMapper(), &setupLog)
```

Comment:
Adding an argument here is not necessary as the function can use the global var declared on L49

Reply:
I believe L49 is declared within main, I have removed the setupLog from globals.  I don't think you can just grab variables up the stack, but maybe I am wrong?

Reply:
Yeah you are right, overlooked the removal.